Tests/README.md

```diff
@@ -16,30 +16,51 @@ python3 -m venv .venv && source .venv/bin/activate
 pip install -r requirements.txt
 ```
 
+At the moment, there are two cloud layers that can be checked: IaaS and KaaS.
+
+### IaaS checks
+
 With a cloud environment configured in your `~/.config/openstack/clouds.yaml`
 and `secure.yaml`, then run
 
 ```shell
-./scs-compliance-check.py scs-compatible.yaml iaas --os-cloud CLOUDNAME
+./scs-compliance-check.py  -a os_cloud=CLOUDNAME -s CLOUDNAME scs-compatible-iaas.yaml
 ```
 
 Replace `CLOUDNAME` with the name of your cloud environment as
 specified in `clouds.yaml`.
 
-The exit code indicates success (0) or failure (!= 0).
+### KaaS checks
+
+Given a kubeconfig file `path/to/kubeconfig.yaml`, run
+
+```shell
+./scs-compliance-check.py -v -a kubeconfig=path/to/kubeconfig.yaml -s SUBJECT scs-compatible-kaas.yaml
+```
+
+Replace `SUBJECT` with an arbitrary, but meaningful subject name.
+Also, please note that the check will always use the `current-context` of the kubeconfig and will
+fail if it isn't set.
+
+For both types of checks, the exit code indicates success (0) or failure (!= 0).
 You can also request a YAML report using the option `-o OUTPUT.yaml`
```

Comment:
this should be moved befre the case distinction, otherwise it may be missed by people who aren't interested in the KaaS casr; maybe introduce a general commandline first (with ellipses for the -a and the spec file)